<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendUI</class>
 <widget class="QWidget" name="SendUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>50</y>
     <width>702</width>
     <height>1034</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="NumLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <widget class="QPushButton" name="num0Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="num5Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="num9Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="num1Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="num4Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="num2Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="clearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="num3Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="num7Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="num6Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="num8Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="eraseButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Erase</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="myLineEdit" name="budgetlineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>610</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>예금주명</string>
   </property>
  </widget>
  <widget class="myLineEdit" name="targetUIDlineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>520</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mynamelineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UIDlabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>상대 계좌번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="budgetlabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>보낼 금액</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>1081</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>송금입니다. 원하시는 계좌번호를 입력해주세요</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="myUIDlineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UIDlabel_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>내 계좌번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mybudgetlineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="budgetlabel_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>내 잔고</string>
   </property>
  </widget>
  <widget class="QLabel" name="UIDlabel_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>읽기 전용</string>
   </property>
  </widget>
  <widget class="QLabel" name="UIDlabel_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>쓰기 가능</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>720</y>
     <width>450</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>720</y>
     <width>450</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
